### DISCUSSION TITLE
minutes above threshold HR

### DISCUSSION THREAD

**Discussion by sloth201** [tcgoetz/GarminDB#184] [Q&A]
Hello,
I'd like to calculate a daily parameter which returns the number of minutes per day spent above a user-defined heart rate. Has anyone done something similar?
Thanks

**Comment by sloth201** [comment]
I see that the parameters "moderate_activity_time" and "vigorous_activity_time" exist in the daily summary db, so these do approximately what I need.

**Comment by tcgoetz** [comment]
activity time is Garmin's version of cardio minutes.
